src/Narochno.Primitives/project.json

```diff
@@ -8,11 +8,12 @@
     "projectUrl": "https://github.com/Narochno/Narochno.Primitives"
   },
 
-  "dependencies": {
-    "System.Collections": "4.3.0",
-    "System.Reflection.Extensions": "4.3.0",
-    "System.Reflection.TypeExtensions": "4.3.0"
-  },
+    "dependencies": {
+        "System.Collections": "4.3.0",
+        "System.Linq": "4.3.0",
+        "System.Reflection.Extensions": "4.3.0",
+        "System.Reflection.TypeExtensions": "4.3.0"

```

Comment:
Hm yes. CI is still not silky smooth, should investigate when one of us gets time